NetUtils/Interface.swift

```diff
@@ -1,11 +1,23 @@
 //  Copyright (c) 2015 Stefan van den Oord. All rights reserved.
 
 import Foundation
+#if os(Linux)
+    import Glibc
+    typealias castAfInet = UInt16
```

Comment:
`castAfInet` sounds like a action (=method). I feel `Family` would be a better name.